### PR TITLE
Minor UI fixes in popup UI

### PR DESCRIPTION
**Pull request by rugk**
Not tested, but it should work.

smallrectangle

### CODE REVIEW

**Comment by rugk** [comment]
Does anyone want to merge this? @toberndo

**Comment by toberndo** [comment]
> $('#uploadEmbeddedBtn, #addFileInput', '#editor_digital_signature').hide();

Looks like you hide the signature field, but is it then displayed in the integrated version of the editor? Please always test against: https://demo.mailvelope.com/manual

> "	I think this should be like this"

Please provide meaningful commit messages

**Comment by rugk** [comment]
[**@toberndo**](https://github.com/toberndo) commented on [22. Dez. 2015, 18:43 MEZ](https://github.com/mailvelope/mailvelope/pull/359#issuecomment-166686088 "2015-12-22T17:43:16Z - Replied by Github Comment Enhancer"):
> > $('#uploadEmbeddedBtn, #addFileInput', '#editor_digital_signature').hide();
> 
> Looks like you hide the signature field, but is it then displayed in the integrated version of the editor? Please always test against: [https://demo.mailvelope.com/manual](https://demo.mailvelope.com/manual)

Ah thanks. Did not knew this.

> > " I think this should be like this"
> 
> Please provide meaningful commit messages

Well.. yeah, sorry. :satisfied:

File: common/ui/editor/tpl/editor-body.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <button id="uploadEmbeddedBtn" class="btn btn-default btn-upload-embedded"><span class="glyphicon glyphicon-paperclip" style="transform: rotate(180deg);"></span>&nbsp;<span data-l10n-id="upload_attachment"></span></button>
         <input type="file" id="addFileInput" multiple="multiple" />
       </div>
-      <div class="pull-right">
+      <div class=""form-group pull-right">
```
Comment (jochenonline) [inline common/ui/editor/tpl/editor-body.html]: two " cannot be right.

Comment (rugk) [inline common/ui/editor/tpl/editor-body.html]: Indeed. I changed it, but I still have not tested it.

**Comment by toberndo** [comment]
This is fixed in v1.6.0

**Comment by rugk** [comment]
Nice! 😄

**Pull request closed without merging**